src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/language/LocalLanguageDetector.scala

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.language
+
+import com.microsoft.partnercatalyst.fortis.spark.logging.FortisTelemetry
+import com.optimaize.langdetect.LanguageDetectorBuilder
+import com.optimaize.langdetect.ngram.NgramExtractors
+import com.optimaize.langdetect.profiles.LanguageProfileReader
+import com.optimaize.langdetect.text.CommonTextObjectFactories
+
+@SerialVersionUID(100L)
+class LocalLanguageDetector extends LanguageDetector {
+  @transient private lazy val languageProfiles = new LanguageProfileReader().readAllBuiltIn
+  @transient private lazy val languageDetector = LanguageDetectorBuilder.create(NgramExtractors.standard()).withProfiles(languageProfiles).build()
+  @transient private lazy val textObjectFactory = CommonTextObjectFactories.forDetectingOnLargeText()
```

Comment:
We don't only have Tweets but also news text, Facebook posts, comments, etc. so overall the default detector is probably fine. We can also run it through both detectors to potentially increase accuracy.